docs/configuration/integrations/integrations-next/app-o11y-receiver-config.md

```diff
@@ -0,0 +1,118 @@
++++
+title = "app_o11y_config"
++++
+
+# app_o11y_receiver_config
+
+The `app_o11y_receiver_config` block configures the `app-o11y-receiver`
+integration. This integration exposes a http endpoint that can receive telemetry
+from the [grafana javascript agent](https://github.com/grafana/grafana-javascript-agent)
```

Comment:
nit: I don't want to bog this down with a full grammar review, but I think any mention of Grafana Javascript Agent should be capitalized like it is in the project.

Reply:
- [ ] captialize